Gems/PhysX/Code/Editor/DebugDraw.h

```diff
@@ -38,7 +38,8 @@ namespace PhysX
         class DisplayCallback
         {
         public:
-            virtual void Display(AzFramework::DebugDisplayRequests& debugDisplayRequests) const = 0;
+            virtual void Display(const AzFramework::ViewportInfo& viewportInfo,
+                AzFramework::DebugDisplayRequests& debugDisplay) const = 0;
```

Comment:
nit/style: `clang-format` will always move parameters to the next line if they won't fit on one (helps with alignment and readability). Might be worth running it over this file (or just this modified code) to see what it does. You can do this with a selection and Ctrl-K, F (repeated in other signatures)